src/SWP/Component/TemplatesSystem/Gimme/Context/Context.php

```diff
@@ -371,4 +371,15 @@ public function restoreTemporaryUnset($id)
 
         return true;
     }
+
+    /**
+     *  Resets context data.
+     */
+    public function reset()
```

Comment:
is this method used somewhere besides the tests?

Reply:
Not yet, but i couldn't find better way beside moving assertions to separate tests.